node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -59,19 +56,20 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val timeEffect: Time[Task]                     = effects.time
   implicit val metricsEffect: Metrics[Task]               = effects.metrics
   implicit val inMemoryPeerKeysEffect: KeysStore[Task]    = effects.remoteKeysKvs(remoteKeysPath)
+  val net                                                 = new UnicastNetwork(src)
```

Comment:
Let's keep the implicits together. Please move it somewhere else.

Reply:
It's part of the implementation of both TranportLayer and NodeDiscovery, that's why it's here